lib/Slider/Slider.js

```diff
@@ -269,19 +269,14 @@ module.exports = kind(
 	* @private
 	*/
 	jumpWrapperComponents: [
-		{name: 'buttonLeft', kind: IconButton, backgroundOpacity: 'transparent', classes: 'moon-slider-button left', icon: 'arrowlargeleft', onSpotlightKeyDown: 'configureSpotlightHoldPulse', onSpotlightSelect: 'previous', ondown: 'downLeft', onholdpulse: 'holdLeft', ondragstart: 'preventDrag', defaultSpotlightDisappear: 'buttonRight'},
+		{name: 'buttonLeft', kind: IconButton, backgroundOpacity: 'transparent', classes: 'moon-slider-button left', icon: 'arrowlargeleft', onSpotlightKeyDown: 'showKnobStatus', onSpotlightKeyUp: 'hideKnobStatus', onSpotlightSelect: 'previous', ondown: 'downLeft', onup: 'hideKnobStatus', onholdpulse: 'holdLeft', ondragstart: 'preventDrag', defaultSpotlightDisappear: 'buttonRight'},
```

Comment:
Hi @DavidUM, I was referring to the fact that pressing down on any key would trigger `showKnobStatus`, regardless of what key it is, because it is assigned to be the handler for `onSpotlightKeyDown` events. I realize we need to handle all of these events - I tried to consolidate the handlers as much as possible so that we only have one handler (`jumpButtonTriggered`) for "start" events, and then we use `hideKnobStatus` for the "end" events. Let me know what you think: https://github.com/enyojs/moonstone/compare/2.6.0-dev...ENYO-2123-aarontam.